OJP/OJP_Trips.xsd

```diff
@@ -813,14 +860,25 @@
 			</xs:element>
 			<xs:element name="Count" type="xs:positiveInteger" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>Number how often the access feature occurs in this PathLink</xs:documentation>
+					<xs:documentation>Number indicating how often the access feature occurs in this PathLink</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="AccessFeatureStatus" type="AccessFeatureStatusEnumeration" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Whether the access feature is available or out of service.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="AccessFeatureStatusText" type="InternationalTextStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Textual information about reduced availability of the access feature, in particular if AccessFeatureStatus is partiallyAvailable.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 			<xs:element name="AccessibilityFeature" type="AccessibilityFeatureTypesEnumeration" minOccurs="0" maxOccurs="unbounded">
 				<xs:annotation>
-					<xs:documentation>Presence of an accessibility feature on the PathLink.</xs:documentation>
+					<xs:documentation>Reference to a situation that affects the availability of the access feature.</xs:documentation>
```

Comment:
The removed documentation should be reintroduced, "Presence..." belongs to *AccessibilityFeature*. "Reference..." should be moved to *SituationFullRef* below.

Reply:
another merge mixup. sorry. fixed it now